tests/nexus_test.py

```diff
@@ -66,6 +66,20 @@ def test_nxobject_root(nexus_group: Tuple[Callable, LoadFromNexus]):
         assert set(root.keys()) == {'entry', 'monitor'}
 
 
+def test_nxobject_items(nexus_group: Tuple[Callable, LoadFromNexus]):
+    resource, loader = nexus_group
+    with resource(builder_with_events_monitor_and_log())() as f:
+        root = nexus.NXroot(f, loader)
+        items = root.items()
```

Comment:
```suggestion
        items = list(root.items())
```
If you accept my other suggestion.